### ISSUE TITLE
mark iocp as something better than "extremely experimental" in the documentation

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3225]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3225|
|Type|enhancement|
|Created|2008-05-06 16:42:03Z|

Since the recent rewrite, iocpreactor is very nearly as working as the supported reactors.  It has only a few failing tests on [buildbot](http://buildbot.twistedmatrix.com/waterfall?builder=win32-py2.4-iocp&builder=winxp32-py2.5-iocp&builder=vista-py2.5-iocp&builder=server-2k8-x86-py2.5-iocp&branch=trunk), the only real deficiency is SSL support.

I suggest pointing out these shortcomings in the docs and dropping the "extremely experimental" bit.

<details><summary>Searchable metadata</summary>

```
trac-id__3225 3225
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1210092123000000 1210092123000000
changetime__1210551272000000 1210551272000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun removed owner|
|-|-|

Here's a proposed doc change:

```
Index: doc/core/howto/choosing-reactor.xhtml
===================================================================
--- doc/core/howto/choosing-reactor.xhtml       (revision 23525)
+++ doc/core/howto/choosing-reactor.xhtml       (working copy)
@@ -146,8 +146,9 @@

     <p>
     Windows provides a fast, scalable event notification system known as IO
-    Completion Ports, or IOCP for short.  An extremely experimental reactor
-    based on IOCP is provided with Twisted.
+    Completion Ports, or IOCP for short.  Twisted includes a reactor based
+    on IOCP which is nearly complete.  The reactor has a handful of known
+    bugs and lacks SSL support.
     </p>

 <pre class="python">
```

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_2"></a>|@therve set owner to @exarkun|
|-|-|

Please apply.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

(In [23542]) Apply doc patch changing status of iocpreactor

Author: exarkun
Reviewer: therve
Fixes: [#3225](https://github.com/twisted/twisted/issues/3225)

Change the language about IOCP reactor from somewhat discouraging
to somewhat encouraging.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
